RecoBTag/Configuration/python/RecoBTag_cff.py

```diff
@@ -79,3 +85,17 @@
     pfCTaggingTask
 )
 btagging = cms.Sequence(btaggingTask)
+
+
+
+# Create b-tagging sequences without pixelClusterTagInfos
+pfBTaggingTaskNoPixelCluster = pfBTaggingTask.copy()
+pfBTaggingTaskNoPixelCluster.remove(pixelClusterTagInfos)
+
+btaggingNoPixelCluster = btagging.copy()
+btaggingNoPixelCluster.remove(pixelClusterTagInfos)
+
+## FastSim modifier (do not run pixelClusterTagInfos because the siPixelCluster input collection is not produced in FastSim)
+from Configuration.Eras.Modifier_fastSim_cff import fastSim
+fastSim.toReplaceWith(btagging, btaggingNoPixelCluster)
```

Comment:
wouldn't it be easier to short-circuit the ```pixelClusterTagInfos``` module configuration than manipulate the sequences?
Also, if these \*NoPixelCluster variants are not meant to be exported, better prefix with ```_```

Reply:
Not sure what short-circuit would mean here. The *NoPixelCluster are actually exported, for instance in miniAOD_tools.py for Era modifiers. Simply adding the underscore instead of  NoPixelCluster would make the name less clear (in my opinion).

Reply:
@zucchett , I think that the suggestion from @slava77 was to apply the modifiers to the pixelClusterTagInfos module, and not to the main reco BTag sequence (or miniAOD).
You can use modifiers to replace pixelClusterTagInfos with some dummy/empty module inside RecoBTag/PixelCluster/python/pixelClusterTagInfos_cfi.py, and you don't have to apply modifiers elsewhere

Reply:
Ok, i can short-circuit the PixelCluster producer by setting a very high threshold on jet pT, so an empty collection is produced and does not fail because it never looks for the siPixelCluster collection. I moved the modifiers to pixelClusterTagInfos_cfi.py, and removed them from the b-tagging sequence and partially from miniAOD_tools.py. However in the latter i still have to avoid that the code looks for the pixelClusterTagInfo collection, so some modifiers on the modules in the miniAOD sequnce are still needed.